pkg/operator/starter.go

```diff
@@ -152,6 +153,8 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 		statusControllerOptions = append(statusControllerOptions, apiservercontrollerset.WithStatusControllerPdbCompatibleHighInertia("APIServer"))
 	}
 
+	infraLister := configInformers.Config().V1().Infrastructures().Lister()
```

Comment:
unused?

Reply:
Used in `guard.IsSNOCheckFnc(infraLister)()` on lines 223 and 230

Reply:
ah, sorry, it is used on line 223 and 230

Reply:
do we have to wait for `infraLister` to be initialized?

Reply:
Initialized at https://github.com/openshift/cluster-openshift-apiserver-operator/blob/master/pkg/operator/starter.go#L319

Reply:
Extending the create/delete functions with `infraInformer.HasSynced()` check.